src/System.Net.Security/tests/FunctionalTests/ClientAsyncAuthenticateTest.cs

```diff
@@ -139,8 +139,7 @@ private Task ClientAsyncSslHelper(SslProtocols clientSslProtocols, SslProtocols
                 using (SslStream sslStream = new SslStream(client.GetStream(), false, AllowAnyServerCertificate, null))
                 {
                     Task async = sslStream.AuthenticateAsClientAsync("localhost", null, clientSslProtocols, false);
```

Comment:
Can we change the variable name from 'async' to something like 'task' or 't'?  Seeing 'async used as a variable name instead of a keyboard is confusing.  Even the code-coloring formatter thinks it is a keyword.